.dockerignore

```diff
@@ -1,3 +1,2 @@
-apps
```

Comment:
I would prefer having this folder (`app`) ignored to prevent developers for releasing private apps.

Apps can still be used from containers thanks to [docker --volumes](https://docs.docker.com/storage/volumes/). I guess this requires additional minor changes in `doc/usage.md` and `run.sh`.

Reply:
✔restored .dockerignore
✔doc/usage.md
✔docker-compose.yml
✘ run.sh (no changes required, the whole apps/ directory is mounted)